perfkitbenchmarker/scripts/object_storage_api_tests.py

```diff
@@ -518,6 +593,148 @@ def SingleStreamThroughputBenchmark(storage_schema, host_to_connect=None):
                   host_to_connect=host_to_connect)
 
 
+def MultiStreamThroughputBenchmark(storage_schema, host_to_connect=None):
+  """A benchmark test for single stream upload and download throughput.
+
+  Args:
+    storage_schema: The schema of the storage provider to use, e.g., "gs"
+    host_to_connect: An optional host endpoint to connect to.
+
+  The object size is given by FLAGS.object_size_kb. The total number
+  of objects to transfer (summed across all streams) is given by
+  FLAGS.iterations. The benchmark starts at FLAGS.write_start_time, if
```

Comment:
Oops, I used that flag in an earlier version of this code before replacing it with `FLAGS.objects_per_stream`. Fixed.